packages/gatsby-starter-ts/src/components/seo.tsx

```diff
@@ -93,4 +94,37 @@ const SEO: React.FC<Props> = ({
   )
 }
 
+export const someTestQuery = gql`
```

Comment:
I think it would be beneficial to add comment here explaining what this is.



```suggestion
/**
 * `gatsby-plugin-ts`/`gatsby-plugin-graphql-codegen` supports codegen for
 * non-gatsby schemas. This is useful if you're using a client-side GraphQL client.
 *
 * Read more at: https://github.com/d4rekanguok/gatsby-typescript/blob/master/packages/gatsby-plugin-graphql-codegen/readme.md#additional-schema-options-for-optionsadditionalschemas
 */
export const someTestQuery = gql`
```